HTML/HTML.sublime-syntax

```diff
@@ -280,7 +280,7 @@ contexts:
     - include: else-pop
 
   doctype-content-type:
-    - match: (?:PUBLIC|SYSTEM)\b
+    - match: (?i)(?:PUBLIC|SYSTEM)\b
```

Comment:
Why not the shorter `(?i:PUBLIC|SYSTEM)\b` ??

Reply:
for coding style

Reply:
Most tags use the `(?i: ...)` style.

Reply:
... ok. A bit a mess.

Reply:
changed to `(?i: ...)` style